lib/tasks/test.rake

```diff
@@ -1,52 +1,54 @@
 require 'iiifc'
 
 namespace :test do
-  desc 'Check html'
-  task :html do
-    opts = {
-      check_html: true,
-      assume_extension: true,
-      validation: {
-        report_mismatched_tags: true,
-        report_invalid_tags: true
-      },
-      checks_to_ignore: ['LinkCheck']
-    }
-    HTMLProofer.check_directory(SITE_DIR, opts).run
-  end
+  # desc 'Check html'
+  # task :html do
+  #   opts = {
+  #     check_html: true,
+  #     assume_extension: true,
+  #     validation: {
+  #       report_mismatched_tags: true,
+  #       report_invalid_tags: true
+  #     },
+  #     checks_to_ignore: ['LinkCheck']
+  #   }
+  #   HTMLProofer.check_directory(SITE_DIR, opts).run
+  # end
 
   namespace :links do
     desc 'Check for internal link errors'
     task :internal do
       puts 'Checking for internal link errors'
       opts = {
-        checks_to_ignore: ['ImageCheck', 'HtmlCheck', 'ScriptCheck'],
+        checks: ['Links'],
         disable_external: true,
-        internal_domains: ['localhost:4000']
+        enforce_https: false,
+        swap_urls: { %r{^http\://localhost\:4000} => 'https://iiif.io' },
       }
       HTMLProofer.check_directory(SITE_DIR, opts).run
     end
 
-    desc 'Check for *iiif.io* link errors'
-    task :iiif do
-      puts 'Checking for link errors in *iiif.io* sites'
-      opts = {
-        checks_to_ignore: ['ImageCheck', 'HtmlCheck', 'ScriptCheck'],
-        url_ignore: [/^((?!iiif\.io).)*$/, 'github'] # temporarily ignore iiif.io github repo errors
-      }
-      HTMLProofer.check_directory(SITE_DIR, opts).run
-    end
+    # desc 'Check for *iiif.io* link errors'
```

Comment:
What is this test doing that is not covered by either the internal link checker, or the external link checker?